### PR TITLE
add TER

### PR DESCRIPTION
**Pull request by monsieurzhang**

### CODE REVIEW

File: utilities/score/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,5 @@
 FROM ubuntu:16.04
+FROM python:3
```
Comment (guillaumekln) [inline utilities/score/Dockerfile]: Why is this needed?

Comment (monsieurzhang) [reply utilities/score/Dockerfile]: It reports an error for python2, like
`File "entrypoint.py", line 63, in eval_TER
    file_tgt.write(('%s\t(%d-)\n' % (line.rstrip(), i)).encode('utf-8'))
UnicodeDecodeError: 'ascii' codec can't decode byte 0xef in position 0: ordinal not in range(128)
`
I tried but seams we have to use **io.open()** with 'utf-8' encoding 
which is a little bit complex.
